public/js/office.web.js

```diff
@@ -23,83 +20,83 @@ $(() => {
       auth2.signOut().then(() => {
         matrixProfile.terminate();
         auth2.disconnect();
-        window.location = '/';
+
+        redirectToHome();
       });
     });
   }
 
-
   function removeUser(userId) {
     $(`#${userId}`).remove();
   }
 
   function showUserInRoom(user, room) {
+    let userView = $(`#${user.id}`);
 
-    var userView = $(`#${user.id}`).length;
-    if (userView == 0) {
+    if (!userView.length) {
       userView = $(`<div  id="${user.id}" class="thumbnail user-room"><img user-presence class="rounded-circle" style="margin:2px;display:flex;" user-id="${user.id}" title="${user.name}" width="50px" src="${user.imageUrl}"></div>`);
-     } else {
-       userView = $(`#${user.id}`).detach();
+    } else {
+      userView.detach();
     }
```

Comment:
Maybe you can invert this check, to avoid if with negative condition.

Reply:
Make sense.